distribution/openrct2.d.ts

```diff
@@ -387,96 +585,211 @@ declare global {
      * APIs for the map.
      */
     interface GameMap {
+        /** The total size of the map. TODO in tiles? */
         readonly size: CoordsXY;
+        /** The number of rides on the game map. */
         readonly numRides: number;
+        /** The number of objects on the game map. */
         readonly numEntities: number;
+        /** An array of all the rides on the map. */
         readonly rides: Ride[];
 
+        /** Get the ride with the given ID. TODO what if doesn't exist?
+         * 
+         * @param id The ride id.
+         * 
+         * @return The ride. TODO what if doesn't exist?
+         */
         getRide(id: number): Ride;
+
+        /**
+         * Get the tile at the given position. TODO what if OOB?
+         * 
+         * @param x The x-coordinate of the tile.
+         * @param y The y-coordinate of the tile.
+         * 
+         * @return The game tile. TODO what if OOB?
+         */
         getTile(x: number, y: number): Tile;
+
+        /**
+         * Get the object with the given id. TODO what if doesn't exist?
```

Comment:
entity

Reply:
Returns null if it doesn't exist.

Reply:
@IntelOrca I assume that's the same for most/all OOB cases?